persistence/src/main/java/edu/unc/lib/dl/persist/services/transfer/FSToFSTransferClient.java

```diff
@@ -112,6 +108,16 @@ public URI transfer(PID binPid, URI sourceFileUri, boolean allowOverwrite) {
                 // Ignore. New file is already in place
                 log.warn("Unable to delete {}. Reason {}", oldFilePath, e.getMessage());
             }
+
+            // Remove source file after deposited
+            if (!source.isReadOnly()) {
```

Comment:
We don't want to perform cleanup in here, the code calling the transfer client should now be responsible for cleanup of the source file. For instance, with deposits the cleanup deposit job should handle this (you should verify that it does so after these changes. To test on your vm I suggest setting the cleanup time property for the deposit application to something very low, like 20second). The intention of switching to copy is so that we don't lose the source data in the case that the larger operation fails.

You should also check if there are other uses of this `transfer` method from non-deposit related activities, since they should now be doing their own source file cleanup.